src/test/java/org/elasticsearch/gateway/RecoveryFromGatewayTests.java

```diff
@@ -435,6 +439,65 @@ public void testReusePeerRecovery() throws Exception {
 
     @Test
     @Slow
+    @TestLogging("indices.recovery:TRACE,index.store:TRACE")
+    public void testSyncFlushedRecovery() throws Exception {
```

Comment:
can we merge this test with the previous one where we randomly replace the first full cluster restart with an synced flush...